src/services/aws/usePaginatedDonations.ts

```diff
@@ -107,6 +93,10 @@ export default function usePaginatedDonationRecords<T extends Args>(args: T) {
     query,
     loadNextPage,
     onQueryChange: setQuery,
+    status: originalArgs?.status || "final",
+    setStatus: (status: DonationsQueryParams["status"]) =>
+      //reset page when switching status (changes table source)
+      setParams((prev) => ({ ...prev, status, page: 1 })),
```

Comment:
Don't know what the desired behavior is here, but these changes no longer preserve previously loaded table items (loaded with "load more" button) for Donations table, which is the current behavior on RC-v1.9.2/staging.
Repro:
1. Sign in with account that has > 10 received donations
2. Go to Donations page
3. Wait for received donations to load
4. Click on "load more"
5. wait for additional donations to load
6. switch to "Pending" donations
7. wait for donations to load
8. switch back to "Received"
9. **expected:** previously loaded donations are preserved <- this is behavior on https://staging.better.giving/donations
**actual:** table is reset, additional donations need to be reloaded with "load more" btn

Reply:
I think the previous one is desirable (avoids loading again). Resetting page I guess is unncessarily worrying the transition from `loaded:page-3` to `pending:page-3`

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2853#:~:text=remove%20page%20reset-,00946c4,-use%20recipient%20id

Reply:
The behavior is still altered, the table gets reset after switching tabs.